src/strings/csv/es.csv

```diff
@@ -652,6 +655,7 @@ OUNCES,Onzas
 OUT_PLANTING,Replantando en el exterior
 OUTPLANT,Replantar en el exterior
 OUTPLANTS_REQUIRE_READY_SEEDLINGS,"Las replantaciones en el exterior requieren lotes con plántulas que estén ""listas""."
+OUTSTANDING,Destacado,In the sense of Pending or Unresolved
```

Comment:
@charlie-tf this didn't exactly work.. sigh
getting this PR in anyways for the other strings.. not sure my other PR with renamed key is going to work either. We can follow up when that PR is open